io.github.syntalos.yaml

```diff
@@ -0,0 +1,244 @@
+id: io.github.syntalos
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-23.08
+separate-locales: false
+
+command: syntalos
+rename-icon: syntalos
+rename-desktop-file: syntalos.desktop
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --filesystem=host
+  - --socket=system-bus
+  - --socket=session-bus
+  - --allow=devel
+  - --allow=bluetooth
+  - --env=PATH=/app/bin:/usr/bin:/usr/sbin
+  - --env=GST_PLUGIN_SYSTEM_PATH=/app/lib/gstreamer-1.0:/usr/lib/extensions/gstreamer-1.0:/usr/lib/x86_64-linux-gnu/gstreamer-1.0
+  - --require-version=1.12.0
+
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+
+add-extensions:
+  org.freedesktop.Platform.GL:
+    directory: lib/GL
+    version: '23.08'
+    subdirectories: true
+    add-ld-path: lib
+    merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d
+    download-if: active-gl-driver
+    enable-if: active-gl-driver
```

Comment:
```suggestion
```

This is installed automatically by the runtime

Reply:
There were some issues with that in the past, but this config section has existed for a long while... So, let's drop it and see what happens!
Thanks for the suggestion!